README.md

```diff
@@ -37,7 +37,7 @@ Upon completion, a CNM Response messages can optionally be sent to a previously
 ## Specification
 There are two types of messages valid in the CNM. The first message is the notification of new data to a DAAC. The second is the optional response message sent from a DAAC to a SIPS.
 
-See https://git.earthdata.nasa.gov/projects/CUMULUS/repos/cumulus-sns-schema/browse/cumulus_sns_schema.json for an up-to-date JSON Schema.
```

Comment:
I do not what was between "https://git.earthdata.nasa.gov/projects/CUMULUS/repos/cumulus-sns-schema/browse/cumulus_sns_schema.json" and "for", but it is not a space character, which leads to a broken link in Readme.